### PR TITLE
feat(traffic_light_classifier): add parameters to launch file for using different tensorrt models

### PR DESCRIPTION
**Pull request by Zeysthingz**
## Description
Added some usefull parameters  to launch file for making easier to use different models on traffic light classification node.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/traffic_light_classifier/src/cnn_classifier.cpp
Comment (yukkysaito) [inline perception/traffic_light_classifier/src/cnn_classifier.cpp:44]: I  think it doesn't need to be a private variable.

File: perception/traffic_light_classifier/src/cnn_classifier.cpp
Comment (yukkysaito) [inline perception/traffic_light_classifier/src/cnn_classifier.cpp:45]: I think it doesn't need to be a private variable.
```suggestion
  input_name = node_ptr_->declare_parameter("input_name", std::string("input_0"));
  output_name = node_ptr_->declare_parameter("output_name", std::string("output_0"));
```

File: perception/traffic_light_classifier/src/cnn_classifier.cpp
Comment (yukkysaito) [inline perception/traffic_light_classifier/src/cnn_classifier.cpp:49]: related link : https://github.com/autowarefoundation/autoware.universe/pull/2794/files#r1093340555
```suggestion
  trt_ = std::make_shared<Tn::TrtCommon>(model_file_path, precision, input_name, output_name);
```

File: perception/traffic_light_classifier/include/traffic_light_classifier/cnn_classifier.hpp
Comment (yukkysaito) [inline perception/traffic_light_classifier/include/traffic_light_classifier/cnn_classifier.hpp:110]: related link : https://github.com/autowarefoundation/autoware.universe/pull/2794/files#r1093340555
```suggestion
```

**Comment by yukkysaito** [comment]
Thank you for your PR 👍 I reviewed and commented

**Review by kaancolak — COMMENTED** [review]
LGTM

**Review by kaancolak — APPROVED** [review]

**Comment by mitsudome-r** [comment]
@yukkysaito Do you have another review? Otherwise, we would like to merge this once @Zeysthingz resolves the conflicts.

File: perception/traffic_light_classifier/launch/traffic_light_classifier.launch.xml
Comment (yukkysaito) [inline perception/traffic_light_classifier/launch/traffic_light_classifier.launch.xml:27]: Could you write to README about this parameters?

Comment (Zeysthingz) [reply perception/traffic_light_classifier/launch/traffic_light_classifier.launch.xml:27]: Yes sure .

**Review by yukkysaito — APPROVED** [review]
LGTM

**Comment by kaancolak** [comment]
Hello @wep21 Daisuke-san,  Could you review this PR?

File: perception/traffic_light_classifier/utils/trt_common.cpp
Comment (wep21) [inline perception/traffic_light_classifier/utils/trt_common.cpp:47]: Are these two empty lines necessary?

**Comment by wep21** [comment]
Could you resolve some conflicts?

File: perception/traffic_light_classifier/utils/trt_common.cpp
Comment (Zeysthingz) [reply perception/traffic_light_classifier/utils/trt_common.cpp:47]: No they are not

**Review by wep21 — APPROVED** [review]
LGTM

**Review by kaancolak — APPROVED** [review]

**Pull request merged**
